package.json

```diff
@@ -37,29 +37,38 @@
 	"dependencies": {
 		"ansicolors": "0.3.2",
 		"app-module-path": "2.2.0",
-		"axios": "0.16.2",
+		"axios": "^0.16.2",
```

Comment:
Due to several problems with Node.js based implementations of fetch, Axios is used in all the backend code. Fetch (native implementation) is used for the frontend with only a polyfill provided from us, reduces the size of the Webpack bundle compared to using Axios there.